configs/taco/ND.yaml

```diff
@@ -7,7 +7,23 @@ links:
     page.manualWait();
     await page.waitForDelay(150000);
     page.done();
-  requestSettings: 
+  requestSettings:
     maxWait: 180000
   message: waiting for 150 seconds to load ND
 
+- name: secondary
+  url: https://app.powerbigov.us/view?r=eyJrIjoiZDcwNDA0ODAtZWU0OC00NDhmLWIzYTMtOTBmZDBkNjNkYTg3IiwidCI6IjJkZWEwNDY0LWRhNTEtNGE4OC1iYWUyLWIzZGI5NGJjMGM1NCJ9&pageName=ReportSection023f3fc1641deaab2bb0
+  overseerScript: |
+    await page.waitForDelay(10000);
+    await page.waitForSelector("i[title='Next Page']");
+    page.click("i[title='Next Page']");
+    await page.waitForDelay(5000);
+    page.click("i[title='Next Page']");
+    await page.waitForDelay(5000);
+    page.click("i[title='Next Page']");
+    await page.waitForDelay(15000);
+    page.done();;
```

Comment:
extra semicolon, otherwise good to merge :)

Reply:
😅  thank you